recipes/IWSLT22_lowresource/hparams/train_w2v2_st.yaml

```diff
@@ -0,0 +1,197 @@
+# ############################################################################
+# Model: E2E ST with wav2vec 2.0 encoder
+# Encoder: wav2vec 2.0 + linear projection
+# Decoder: Transformer
+# Tokens: unigram (1000)
+# losses: NLL
+# Training: Tamasheq-French corpus
+# Author:  Marcely Zanon Boito, 2022
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 5988
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+debug: False
+output_folder: !ref output/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# root data folder points to 17h version inside the github folder
+root_data_folder: IWSLT2022_Tamasheq_data/taq_fra_clean/
```

Comment:
I like that the data folder hierarchy is splitted by variables !

Should a top-level data folder variable be included with `!PLACEHOLDER` ?
In other recipes, the placeholder is a reminder to users to adjust for their specific environments/paths.